patch/common/wdt.yaml

```diff
@@ -30,3 +30,5 @@ WDT:
       CYCLES_64K:    [5, "- 64K (65536) cycles, ~0.5s"]
       CYCLES_128K:   [6, "- 128K (131072) cycles, ~1s"]
       CYCLES_256K:   [7, "- 256K (262144) cycles, ~2s"]
+      CYCLES_512K:   [8, "- 512K (524288) cycles, ~4s"]
+      CYCLES_1024K:  [9, "- 1024K (1048576) cycles, ~8s"]
```

Comment:
Actually, those values are not in here for a reason ;)  If you look closely, the `WDPL` field is only 3 bit wide, meaning `7`(`0b111`) is the biggest value to fit in there.  The additional values in the datasheet use the `WDPH` bit which is unfortunatly separate from the rest.  This is why I instead listed `512K` and `1024K` in `CYCLES_2K` and `CYCLES_4K` respectively.

But as you doing this change shows, the way I did it was probably too confusing.  If you want to, I suggest renaming `CYCLES_2K` to `CYCLES_2K_512K` and `CYCLES_4K` to `CYCLES_4K_1024K`?

Reply:
Ahhh that makes sense, I did not read the descriptions thoroughly enough.  I'll remove the new lines and make the suggested changes.